woocommerce/woocommerce-gutenberg-products-block
# Add editable filter widget headings

While we don't yet have filter block content toggles and headings (#850) we need a method of adding headings the filter blocks. The plan was to simply insert a header block above each, however, this doesn't allow the heading to be hidden when the block does not render (if there are no filters to display).

In the future we may look at using inner blocks and sharing state somehow, however, the interim solution is to just add a heading element which can be customised in the editor, as well as heading level selection which we have in other blocks.

### Screenshots

![Screenshot 2019-11-13 at 14 26 35](https://user-images.githubusercontent.com/90977/68773460-7da7ea80-0623-11ea-9ba5-f3ac2e4c81a9.png)
![Screenshot 2019-11-13 at 14 35 49](https://user-images.githubusercontent.com/90977/68773462-7ed91780-0623-11ea-836b-db35c4376156.png)

### How to test the changes in this Pull Request:

1. Create a price filter block and test you can edit the heading, heading level, and see the changes on the frontend.
2. Repeat this for attribute filters. You need to select an attribute first.
3. If you change the attribute by editing, the heading should also update to reflect whatever attribute you've chosen.
